### PR TITLE
8430 auto account update suite

### LINKED ISSUE #8430: Start testing AutoAccountUpdateSuite
- [x] updateKeyOnAutoCreatedAccount
- [x] modifySigRequiredAfterAutoAccountCreation

### LINKED ISSUE #9343: AutoAccountUpdateSuite.updateKeyOnAutoCreatedAccount() failed inconsistently with INSUFFICIENT_TX_FEE.
### Description

When run AutoAccountUpdateSuite.updateKeyOnAutoCreatedAccount() by itself, it would pass most time.
When run AutoAccountUpdateSuite, updateKeyOnAutoCreatedAccount() test would fail most time with INSUFFICIENT_TX_FEE.

I debugged into it and found CryptoUpdateHandler fee.networkFee was calculated twice, one during Ingest workflow, one during workflow handler with different values with the same transaction body. I added debug line in cryptoUpdateFees() and commented out in my branch 8430-AutoAccountUpdateSuite. Remove the commented out lines, run the test, you should see the details.

### PR DESCRIPTION
**Pull request by iwsimon**
**Description**:

**Related issue(s)**:

Fixes #8430 
Fixes #9343 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java [hunk 1]
```diff
@@ -409,6 +409,12 @@ public static Fees cryptoUpdateFees(
                 Math.max(explicitAutoAssocSlotLifetime, oldLifetime),
                 newSlotsUsage,
                 Math.max(explicitAutoAssocSlotLifetime, newLifetime));
+        //        var fees = feeCalculator
+        //                .addBytesPerTransaction(baseSize)
+        //                .addRamByteSeconds((rbsDelta + slotRbsDelta) > 0 ? rbsDelta + slotRbsDelta : 0)
+        //                .calculate();
+        //        System.out.println("Fees.networkFee: " + fees.networkFee() + " txBody: " + body);
+        //        return fees;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java:417]: Can we remove these commented lines

Comment (iwsimon) [reply hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java]: Removed.

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM !

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM !

**Pull request merged**